hack/deploy-localhost.patch

```diff
@@ -15,7 +15,7 @@ index d7bf1964..d2a9c614 100644
                fieldPath: spec.nodeName
          - name: KUBELET_DIR
            value: /var/lib/kubelet
--        image: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.8.0
+-        image: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.8.1
 -        imagePullPolicy: Always
 +        image: localhost/security-profiles-operator:latest
```

Comment:
Interesting - is this specific to the release?

Reply:
No, it's a patch used for e2e tests and local deployments.

Reply:
Thanks @saschagrunert.

I imagine this should be removed from the PR, then.

Reply:
sorry accidentally slipped that one in

Reply:
@rhmdnd it should be updated now, thanks